### ISSUE TITLE
Homepage not working as user policy

### ISSUE THREAD

**Issue by departet** [mozilla/policy-templates#937]
FF Version: 102.1 ESR
DC Version: Windows Server 2019

RDS Environment

Hey everyone,

i remember the homepage being a computer configuration thing in the past. After updating FF and the ADMX files yesterday i don't get the "as of firefox 62 only working as computer policy.." warning under user configuration but it's still only working as a computer configuration.
Is there something not working quite right or is the homepage gpo still expected to only work under computer configuration?

Thank you very much in advance

**Comment by mkaply** [comment]
All all of our policies work in both user and computer policy.

But if you have computer policy and user policy set, only computer policy us read. (This is consistent with other Windows applications).

**Comment by mkaply** [comment]
Yeah, I really struggled on the right way to do that. I'll reach out to my contacts and make sure that's still the way.
